### ISSUE TITLE
Nan in loss function

### ISSUE THREAD

**Issue by hieugiaosu** [dmlguq456/SepReformer#2]
Thanks for your great repository. But when i try to train with your base pipeline and model, the loss start to produce NaN.
